test/spec/components/Select.spec.js

```diff
@@ -75,6 +75,60 @@ describe('<Select>', function() {
   });
 
 
+  describe('options', function() {
+
+    it('should render options non-disabled per default', function() {
```

Comment:
Maybe change to _should render options enabled_ (cf. https://github.com/bpmn-io/properties-panel/pull/77/commits/3a66f3c7187c9257f578bdba4268c08ecb009844#diff-8a7dd5b0e7a682b72766dfbe7dc3696aa7ad62696c049e971c0602cf452f62eeR221).

Reply:
Updated it to have a clearer wording (_enabled_ in favor of _non-disabled_)

Reply:
Thanks!